neutrino.go

```diff
@@ -945,6 +953,22 @@ func NewChainService(cfg Config) (*ChainService, error) {
 		permanentPeers = cfg.AddPeers
 	}
 
+	// Adding rest peers to chainservice if spesified in config
+
+	restPeers := cfg.RestPeers
+	if len(restPeers) != 0 {
+		// Iterating thought restpeer defined in the config
+		// and checking if they are reachable.
+		for _, restAddr := range restPeers {
+			_, err := url.Parse(restAddr)
+			if err != nil {
+				log.Debugf("error unable to parse address: %w", err)
+				break
```

Comment:
Are you sure that continue is not better? With break, if the first url is bad, all the elements of the array restPeers are lost.